cast/src/main.rs

```diff
@@ -53,6 +54,10 @@ enum Commands {
 #[tokio::main]
 async fn main() -> Result<()> {
     let cli = Cli::parse();
+    let accounts_file_path = match cli.accounts_file_path {
+        Some(path) => path,
+        None => Utf8PathBuf::from(format!("{}/.starknet_accounts/starknet_open_zeppelin_accounts.json", dirs::home_dir().expect("failed to read the home directory").to_str().expect("failed to convert home directory to string"))),
+    };
 
     // todo: #2052 take network from scarb config if flag not provided
```

Comment:
👀

Reply:
+ any other ideas on what we can extract from scarb config?

Reply:
> 👀

Sorry, could you clarify what's wrong? This emoji doesn't tell me what you mean :(

Reply:
sure, sorry :D there is a todo for the issue this PR addresses, so I guess it should be resolved in this PR

Reply:
Right, sorry, I did not notice, thanks!

Reply:
Yes, I see now, didn't notice, thx

Reply:
[remove resolved todo](https://github.com/software-mansion/protostar/pull/2138/commits/05adf8aabd338e7d877f767f22285de4df2771cc)

Reply:
cool, but we should rather add this functionality in this PR, so let's lleave unresolved until it's done :+1:

Reply:
[pull network from scarb config](https://github.com/software-mansion/protostar/pull/2138/commits/a0f6a1e9f013d57e1a486ff8321ff5d23562d585)